source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -29,9 +31,9 @@ operation in a JSON object as follows:
      toppings: [ "San Marzano tomatoes", "mozzarella di bufala cheese" ],
    };
 
-To insert the document into the collection reference ``pizzaCollection``,
-specify the document as the first parameter of your call to the
-``insertOne()`` method as shown below:
+To insert this document into a collection called ``pizzaCollection``,
+pass ``pizzaDocument`` as the first parameter to the ``insertOne()``
+method as shown below: 
```

Comment:
This is also repeated below. It be worth stating that the examples on this page work with a collection called pizzaCollection, and we'll be using the different methods to insert various documents representing pizza types